examples/distributed_lock/README.md

```diff
@@ -0,0 +1,78 @@
+# Example - Acquire and release distributed locks
+
+This example demonstrates the [Distributed Locks component] APIs in Dapr.
+It demonstrates the following APIs:
+- **try lock**: Attempts to acquire a distributed lock from the lock store.
+- **unlock**: Attempts to release (a previously acquired) distributed lock
+
+It creates a client using `DaprClient`, uses a local lock store defined in
+[`./components/lockstore.yaml`](./components/lockstore.yaml) and invokes
+all the distributed lock API methods available as example.
+
+## Pre-requisites
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started)
+- [Install Python 3.7+](https://www.python.org/downloads/)
+
+## Install Dapr python-SDK
+
+<!-- Our CI/CD pipeline automatically installs the correct version, so we can skip this step in the automation -->
+
+```bash
+pip3 install dapr dapr-ext-grpc
+```
+
+## Run the example
+
+Change directory to this folder:
+```bash
+cd examples/distributed_lock
+```
+
+To run this example, the following code can be utilized:
+
+<!-- STEP
+name: Run state store example
+expected_stdout_lines:
+  - "== APP == INFO:root:Will attempt to acquire a lock for resource=python-sdk-example-lock-resource"
+  - "== APP == INFO:root:This client identifier is client-271c0755-c603-4cea-aa17-88c6292c1d0d"
```

Comment:
Giving this will be used for automated tests, it might be better to remove the UUID component from `client_id`. The same can be said for the the representation for `TryLockResponse` bellow.